src/components/variable.jsx

```diff
@@ -20,6 +25,29 @@ const Variable = React.createClass({
   getTodayInstant() {
     return new Date().toJSON().slice(0, 10)
   },
+  navigateProgramatically(e) {
```

Comment:
No, this seems to be the problem: https://stackoverflow.com/questions/32261441/component-does-not-remount-when-route-parameters-change